erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -263,6 +264,94 @@
                         **_ERIC_GLOBAL_INITIALISATION_ERRORS, **_ERIC_GLOBAL_ERRORS, **_ERIC_TRANSFER_ERRORS,
                         **_ERIC_CRYPT_ERRORS, **_ERIC_IO_ERRORS, **_ERIC_PRINT_ERRORS}
 
+_ERIC_ERRORS_NO_RETRY = ["ALREADY_OPEN_UNLOCK_CODE_REQUEST", "ELSTER_REQUEST_ID_UNKNOWN", "ERIC_GLOBAL_PRUEF_FEHLER",
```

Comment:
I think this is a lot and probably will never be updated. I have two suggestions:
1. Reverse this. Only name errors that we do want retried. - that list is potentially shorter?
2. Only name those errors that we know will occur and do not want retried: "ALREADY_OPEN_UNLOCK_CODE_REQUEST", "ELSTER_REQUEST_ID_UNKNOWN", "ERIC_GLOBAL_PRUEF_FEHLER", "ERIC_GLOBAL_BUFANR_UNBEKANNT", "INVALID_BUFA_NUMBER", "INVALID_TAX_NUMBER", "ERIC_GLOBAL_STEUERNUMMER_UNGUELTIG" and most importantly: "ERIC_TRANSFER_ERR_XML_NHEADER"
3. Reference the complete class of error (like `_ERIC_GLOBAL_ERRORS`) and only remove those that we do want retried - which is basically 1. but different^^

I think I'd probably prefer 2 for simplicity and readability, but we probably had that discussion about which errors to retry for a reason. :x

Reply:
You are right, I thought it would be better to name those errors that do not need a retry because we had in the confluence list "unknown errors" also listed for a retry, but those errors will be automatically retried since in the perform_job we are only catching "EricProcessNotSuccessful"
This is why I reversed the list and added those listed in confluence that should be retried and yes! The list is much shorter